.coderabbit.yaml

```diff
@@ -13,8 +31,17 @@ reviews:
         compatibility, and expressiveness.
     - path: '**/*.sh'
       instructions: >-
-          Review the shell scripts, point out issues relative to security,
-          performance, and maintainability.
+        Review the shell scripts, point out issues relative to security,
+        performance, and maintainability.
+
   auto_review:
+    drafts: false
+    # disables review for new commits
+    auto_incremental_review: false
```

Comment:
I think the incremental reviews are good?

Reply:
Yeah, @swift1337  do you have an example where this is not relevant to keep it?
Can we try first with the new directive in the config to see?

Reply:
Finally approving it. I think we can ask review manually if it is relevant.

Reply:
I see a lot of redundant follow up of new commits. Indeed let's ask for incremental reviews manually.

<img width="817" alt="image" src="https://github.com/user-attachments/assets/4f3113b5-9962-4169-bad5-98225f8eee3a">
<img width="841" alt="image" src="https://github.com/user-attachments/assets/067cc77c-8030-4839-904f-c4855478fbad">
<img width="849" alt="image" src="https://github.com/user-attachments/assets/4902168b-293b-412c-baf1-1118cedf41fe">


etc...